README.md

```diff
@@ -16,9 +16,9 @@ This repo is linked to main BGNN Fish project rpo [BGNN_Snakemake](https://githu
 
 ## 2 - Usage
 
-Input : 
+Input example : 
   + fish image .jpg [see INHS_FISH_000742.jpg](https://github.com/hdr-bgnn/Crop_image/blob/main/Test_data/INHS_FISH_000742.jpg)
-  + Metadata.json [see INHS_FISH_000742.json](https://github.com/hdr-bgnn/Crop_image/blob/main/Test_data/INHS_FISH_000742.json)
+  + Metadata.json [see INHS_FISH_000742.json](https://github.com/hdr-bgnn/Crop_image/blob/new-input-format/Test_data/INHS_FISH_000742.json)
```

Comment:
The above link contains an URL to the JSON file in the `new-input-format` branch.
Once this PR is merged and this branch is deleted the link will no longer work.
You can just provide a relative path that will always point to the file in the current branch.

```suggestion
  + Metadata.json [see INHS_FISH_000742.json](Test_data/INHS_FISH_000742.json)
```

Reply:
@thibaulttabarin I think the request here was to change _all_ such paths accordingly, not just this one.

Reply:
Chnaged to relative path
https://github.com/hdr-bgnn/Crop_image/commit/ae3c32f9620f25c0da6b4c1d3540e4cf95f0e23c